gcp/cloud_run_functions/run_followup_scheduling/main.py

```
@@ -436,9 +453,19 @@ def followup_scheduling_handler(cloud_event: CloudEvent) -> None:
     survey_ids = config.followup_surveys.survey_ids
     sms_template = config.followup_surveys.sms_template
 
+    try:
+        followup_times = get_followup_times(message.work_shift)
+    except KeyError:
+        logger.error(
+            "Unknown work_shift '%s' for response %s -- "
+            "acknowledging to prevent infinite retry",
+            message.work_shift,
+            message.response_id,
+        )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)